### PR TITLE
Some PSVR documentation and one fix

### PR DESCRIPTION
**Pull request by Wallbraker**
* Refactor the code and add some comments so it's easier to understand what the code is doing.
* Use all samples, a previous commit seems to have accidentally hardcoded only to use sample one.
* ~HTC PSVR~ Sony PSVR, oopsie.
* Add TODO about handling startup better after unclean session close.

### CODE REVIEW

File: src/drv_psvr/psvr.c [hunk 1]
```diff
@@ -64,30 +78,34 @@ static void handle_tracker_sensor_msg(psvr_priv* priv, unsigned char* buffer, in
...
 
-	float dt = tick_delta * TICK_LEN;
 	vec3f mag = {{0.0f, 0.0f, 0.0f}};
 
-	for(int i = 0; i < 2; i++){
-		accel_from_psvr_vec(s->samples[0].accel, &priv->raw_accel);
-		gyro_from_psvr_vec(s->samples[0].gyro, &priv->raw_gyro);
+	for (int i = 0; i < 2; i++) {
+		float dt = tick_delta * TICK_LEN;
+		accel_from_psvr_vec(s->samples[1].accel, &priv->raw_accel);
```
Comment (pH5) [inline src/drv_psvr/psvr.c]: This should be `s->samples[i]`, not `s->samples[1]`, like it was before I broke it in f8420910a6ad.

Comment (Wallbraker) [reply src/drv_psvr/psvr.c]: That is fixed in a later commit, I tried to make the refactor commit be completely a none functional change. I see now that I managed to change it from `0` to `1`.

Comment (Wallbraker) [reply src/drv_psvr/psvr.c]: Ugh, I'm stupid... confused 1 and i. :-(

Comment (Wallbraker) [reply src/drv_psvr/psvr.c]: Fixed!

**Comment by Wallbraker** [comment]
Thanks for the review! :)

**Pull request merged**